src/main/kotlin/com/mparticle/kits/ApptentiveKitUtils.kt

```diff
@@ -12,18 +16,27 @@ import com.mparticle.MParticle
 object ApptentiveKitUtils {
     @JvmStatic
     fun registerApptentiveActivityContext(callback: ApptentiveActivityInfo) {
+        val broadcastReceiver = object : BroadcastReceiver() {
+            override fun onReceive(context: Context, intent: Intent) {
+                if (intent.action?.startsWith(MParticle.ServiceProviders.BROADCAST_ACTIVE) == true) {
+                    Apptentive.registerApptentiveActivityInfoCallback(callback)
+                }
+            }
+        }
         if (MParticle.getInstance()?.isKitActive(MParticle.ServiceProviders.APPTENTIVE) == true) {
             Apptentive.registerApptentiveActivityInfoCallback(callback)
+            Log.d("ApptentiveKitUtils", "registerApptentiveActivityContext: kit is active")
```

Comment:
```suggestion
            Logger.debug("ApptentiveKitUtils", "registerApptentiveActivityContext: kit is active")
```

can you please use here Logger.debug

Reply:
Updated.